ms-campus-kid-grades/database.conf

```diff
@@ -0,0 +1,6 @@
+POSTGRES_USER= postgres
```

Comment:
These are not the configurations for the database we already have in the docker-compose, please use those ones.
```
DB_HOST = host.docker.internal
DB_USER = root
DB_PASS = 1234
DB_DATABASE = db_grade
DB_PORT = 4006
```
Be sure that your project works well with them.
You have to run the `docker-compose` file.
There are some instructions in the `README.md` file